pyproject.toml

```
@@ -38,7 +38,7 @@ classifiers = [
 ]
 dependencies = [
   "awkward>=2.5.0",
-  "uproot>=5.1.2",
+  "uproot>=5.2.0rc1",
```

```suggestion
  "uproot>=5.2.0rc2",
```

This can now be updated to the new pre-release!